packages/babel-parser/test/fixtures/es2015/uncategorised/217/output.json

```diff
@@ -20,7 +20,7 @@
           "extra": {
             "rawValue": "\\u{}",
             "raw": "\"\\u{}\"",
-            "expressionValue": "null"
+            "expressionValue": ""
```

Comment:
The new behaviour here is better than current main. Should we set `expressionValue` to be `null` when the string can not be successfully parsed?

Reply:
I admit I don't know why the behavior changed :grimacing: 
But yes, `null` would be even better.

Reply:
Actually, `expressionValue` is a property of directives that represent the `.value` property of the originally parsed `StringLiteral`. I feat that making it `null` would break anything relying on the fact that `StringLiteral.value` is a string, so I prefer the behavior currently implemented by this PR of just replacing invalid escapes with `""` (for example, `ab\u{}cd` is `abcd`).